### ISSUE TITLE
SMS backup not backing up 2-23-2016

### ISSUE THREAD

**Issue by spinncity** [jberkel/sms-backup-plus#623]
SMS backup+ not auto backing up.  It's been working fine for months and all of a sudden it stopped.  It will back up manually but not auto.  I have it set for every 30 minutes and every minute on incoming.  I have a Samsung Galaxy S5 Active.  Any assistance you can offer is greatly appreciated.

Thanks,

Melissa

**Comment by J4259** [comment]
I'm having the same issue since I switched to the Galaxy S6.  Does anyone know how to fix this?  Maybe a specific beta update?

**Comment by jberkel** [comment]
This is probably related: #608

**Comment by Behtaj** [comment]
Every so often it stops backing up. Manual backup won't work. The app thinks it has backed up to a date in the future.

I've had to reinstall the app several times in the past couple years in order to get backups started again. Since there's no way of backing up only the missing days, the app backs up everything, causing duplicates each time.

I've reported this problem via email and got no reply. Today I found this forum, I hope this will initiate a bug fix.

Samsung Note 3
